README.md

```diff
@@ -55,6 +55,7 @@ Reading it in C++ is easy with toml++:
 
 auto config = toml::parse_file( "configuration.toml" );
 
+using namespace std::literals;
```

Comment:
Please move this to the top of the file, just after the #include. Also ensure it has a blank line above and below.